WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/ai/preview/AiProductPreviewScreen.kt

```diff
@@ -174,6 +198,17 @@ private fun ProductPreviewContent(
                 .padding(dimensionResource(id = R.dimen.major_100))
         )
 
+        if (true || state.shouldShowVariantSelector) {
```

Comment:
Possible leftover from testing? 
```suggestion
        if (state.shouldShowVariantSelector) {
```

Reply:
Yes, true, this was a leftover from when I was testing the UI before implementing the real generation logic, thanks for catching it up.